### ISSUE TITLE
Front end: links missing underline style

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#192]
<img width="442" alt="Screen Shot 2023-06-23 at 9 34 31 AM" src="https://github.com/equalizedigital/accessibility-checker/assets/13153456/0b2b7364-6fe8-4277-9536-b132d7dd8eb3">
Accessibility New Window Warnings plugin is a link, but it's missing the underline.

On hover, it is adopting the styles from the website, which increases the chances that there may be a color contrast error. We need to define styles for links in this element that don't rely on color alone.

Hover on RWC:
<img width="415" alt="Screen Shot 2023-06-23 at 9 36 05 AM" src="https://github.com/equalizedigital/accessibility-checker/assets/13153456/577ed351-b207-4424-a2ae-e752388c8422">
